core/modules/replica/ConfigApp.cc

```diff
@@ -431,7 +807,26 @@ int ConfigApp::runImpl() {
 
     string const context = "ConfigApp::" + string(__func__) + "  ";
 
-    _config = Configuration::load(configUrl());
+    if (_command == "SCHEMA_VERSION") {
+        cout << ConfigurationSchema::version << endl;
+        return 0;
+    }
+
+    // Do not attept loading the configuration since the database may not exist
+    // at this time, or it may require special handling.
+    if (_command == "MYSQL_CREATE") return _create();
```

Comment:
Right no that's impossible because a value of `_command` is set by the parser from a list of available commands. The commands are strings typed by a user:
```
qserv-replica-config MYSQL_CREATE <options> <flags>
```

Reply:
I'm not sure why it is impossible:

    const std::string MYSQL_CREATE_CMD = "MYSQL_CREATE";

should work, no? And use this constant when you create command ind do a dispatch, what worries me is that you have may command  names and I cannot verify that your dispatch code matches the definitions.